projects/packages/my-jetpack/_inc/components/product-detail-card/index.jsx

```diff
@@ -122,3 +146,12 @@ export function BackupDetailCard() {
 export function BoostDetailCard() {
 	return <ProductDetailCard slug="boost" />;
 }
+
+/**
+ * BoostDetail component
+ *
+ * @returns {object} BoostDetail react component.
+ */
+export function BoostDetail() {
+	return <ProductDetail slug="boost" />;
+}
```

Comment:
We're using these specific components in other places than storybook?

Reply:
nop

Reply:
I think it could be moved for `stories` then, what do you think?

Reply:
yes, let's remove it.